code/modules/Martial Arts/adminfu.dm

```diff
@@ -0,0 +1,117 @@
+///Adminfu
```

Comment:
comment on this file in general:
please don't introduce new folders that are upper case and contain spaces, please rename it to `code/modules/martial_arts/`, and move martial.dm into this folder